modules/aws-iam-policies/main.tf

```diff
@@ -40,7 +50,14 @@ data "aws_iam_policy_document" "emr_creator_policy" {
   statement {
     effect = "Allow"
     actions = [
-      "elasticmapreduce:Describe*",
+      "elasticmapreduce:DescribeCluster",
```

Comment:
Looking [here](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonelasticmapreduce.html#amazonelasticmapreduce-cluster) appears there are still some opportunities for scoping down.

DescribeStudio -> studio* (do we need?)
DescribeStep -> cluster*
DescribeNotebookExecution -> notebook-execution* (do we need?)
DescribeJobFlows -> cluster*
DescribeEditor -> editor* (do we need?)
DescribeCluster -> cluster*

Reply:
Was able to remove those in question: DescribeStudio, DescribeNotebookExecution, DescribeEditor

Also scoped to named clusters for DescribeStep, DescribeJobFlows, DescribeCluster